app/controllers/cclow/climate_targets_controller.rb

```diff
@@ -10,6 +10,8 @@ def index
       add_breadcrumb('Search results', request.path) if params[:q].present? || params[:recent].present?
 
       @climate_targets = Queries::CCLOW::TargetQuery.new(filter_params).call
+      @admin_panel_section_title = 'Climate targets'
```

Comment:
I would probably encapsulate this in some common method, similar way as `add_breadcrumb` is done.

Reply:
good idea